src/Directory.Packages.props

```diff
@@ -1,12 +1,13 @@
 <Project>
     <PropertyGroup>
         <ManagePackageVersionsCentrally>true</ManagePackageVersionsCentrally>
-        <AkkaVersion>1.5.8</AkkaVersion>
+        <AkkaVersion>1.5.13</AkkaVersion>
```

Comment:
As part of this I bumped the versions to 1.5.13 for Akka and added `Akka.Persistence.Hosting` as there were some hocon extension methods added that were needed